src/elements/content-explorer/RenameDialog.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { Modal, TextInput } from '@box/blueprint-web';
+import type { BoxItem } from '../../common/types/core';
+
+import { ERROR_CODE_ITEM_NAME_TOO_LONG, ERROR_CODE_ITEM_NAME_IN_USE } from '../../constants';
+
+import messages from '../common/messages';
+
+import './RenameDialog.scss';
+
+export interface RenameDialogProps {
+    errorCode: string;
+    isLoading: boolean;
+    isOpen: boolean;
+    item: BoxItem;
+    onCancel: () => void;
+    onRename: (nameWithoutExt: string, extension: string) => void;
+    parentElement: HTMLElement;
+}
+
+/* eslint-disable jsx-a11y/label-has-for */
```

Comment:
is this still needed? if so, could you use next-line instead of disabling the entire component

Reply:
good catch! it is not needed since <label> and <input> are replaced by <TextInput>.